src/main/java/duke/core/TypoCorrector.java

```diff
@@ -20,13 +20,13 @@
 
     //Sets of "Dictionaries" for the command keyword, categorised by number of keywords contain in a supported commands.
     private static final ArrayList<String> oneKeywordCommand = new ArrayList<String>(
-            Arrays.asList("bye"));
+        Arrays.asList("bye", "duke", "help"));
     private static final ArrayList<String> twoKeywordsCommands = new ArrayList<String>(
-            Arrays.asList("list patients", "list tasks"));
+        Arrays.asList("list patients", "list tasks"));
     private static final ArrayList<String> otherCommands = new ArrayList<String>(
-            Arrays.asList("update patient", "update task",
-                    "delete patient", "delete task", "add task", "add patient",
-                    "assign by", "find patient", "find task"));
+        Arrays.asList("update patient", "update task",
+            "delete patient", "delete task", "delete patienttask", "add task", "add patient",
```

Comment:
Parser has been implemented in a way that we don't have to compensate for spacing with keywords such as `patienttask`; in this case, `delete patient task` should work fine.

Reply:
Noted.